tardis/transport/montecarlo/base.py

```diff
@@ -158,12 +160,24 @@ def run(
         self.transport_state = transport_state
 
         number_of_vpackets = self.montecarlo_configuration.NUMBER_OF_VPACKETS
+        number_of_rpackets = len(transport_state.packet_collection.initial_nus)
```

Comment:
We should add a packet count property to the transport state. I'll make an issue- don't worry about it here.